app/src/main/java/org/fossasia/phimpme/utilities/Utils.java

```diff
@@ -163,13 +163,13 @@ public static String getMimeType(String url) {
         }
         return type;
     }
-    public static boolean checkNetwork(Context context, @NonNull View view) {
-        if (isInternetOn(context)) {
-            return true;
-        } else
-            SnackBarHandler.show(view,context.getString(R.string.not_connected));
-        return false;
-    }
+//    public static boolean checkNetwork(Context context, @NonNull View view) {
```

Comment:
@kk63852 instead of creating your own method, use this method which is in utils. It has been created for the purpose of being called from anywhere. Or is there any particular reason for this change you made?